src/ui/public/vislib/visualizations/point_series/line_chart.js

```diff
@@ -169,7 +171,9 @@ export default function LineChartFactory(Private) {
           .interpolate(interpolate)
           .x(isHorizontal ? cx : cy)
           .y(isHorizontal ? cy : cx);
-        return d3Line(data.values);
+        return d3Line(data.values.filter(function (d) {
```

Comment:
will extract this in a separate PR because as you say this is used in many places and i don't want to put too many things in this PR